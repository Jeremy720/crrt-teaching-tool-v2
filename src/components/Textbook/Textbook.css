.Textbook {
  background-color: #FFF;
}

.Textbook-navbar {
  background-color: #f7f7f7;
  height: 3.5rem;
  display: flex;
  align-items: center;
  padding: 0 0 0 1rem;
}

.Textbook-navbar-link, .Textbook-NavLink {
  text-decoration: none;
  font-size: 20px;
  color: #000000e6;
  font-weight: 500;
}

.Textbook-container { 
  padding: 2rem 6rem;
  text-align: left;
}

.Textbook-h2 {
  font-size: 20px;
  color: #000000e6;
}

.Textbook-h3 {
  font-size: 18px;
  color: #000000e6;
}

hr {
  border: 0;
  border-top: 1px solid #0000001a;
  width: 100%;
}

.table {
  overflow-y: scroll;
  overflow-x: scroll;
}

table td, th {
  border-top: none !important;
}

td, th {
  padding: 5px 20px;
  width: 100px;

}

