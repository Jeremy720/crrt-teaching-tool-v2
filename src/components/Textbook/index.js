import React from "react";
import "./Textbook.css";

function Textbook() {
  return (
    <div>
      <h1>CRRT Textbook!</h1>
    </div>
  );
}

export default Textbook;
