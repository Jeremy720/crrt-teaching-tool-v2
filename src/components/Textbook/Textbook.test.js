import React from "react";
import Textbook from "./";
import { shallow } from "enzyme";

describe("Textbook", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Textbook />);
  });

  it("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
