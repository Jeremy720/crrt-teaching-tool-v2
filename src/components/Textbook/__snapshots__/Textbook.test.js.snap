// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textbook matches the snapshot 1`] = `ShallowWrapper {}`;
